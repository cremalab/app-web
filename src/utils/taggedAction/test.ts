import { taggedAction } from "."

describe("taggedAction", () => {
  it("returns a `TaggedAction<Type, Payload>`", () => {
    // Arrange
    const action = { type: "SOME_TYPE", payload: 2 }

    // Act
    const received = taggedAction(action)

    // Assert
    expect(received).toMatchSnapshot()
  })
})
