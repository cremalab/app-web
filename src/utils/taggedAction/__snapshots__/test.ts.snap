// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`taggedAction returns a \`TaggedAction<Type, Payload>\` 1`] = `
Object {
  "data": Object {
    "payload": 2,
    "type": "SOME_TYPE",
  },
  "tag": "SOME_TYPE",
}
`;
